resources/cluster-essentials/files/crd-kubeless.yaml

```diff
@@ -1,6 +1,5 @@
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This CRD no longer exists in Kyma, it has been removed.

Reply:
thx I will update the PR